CMakeLists.txt

```diff
@@ -443,7 +443,9 @@ target_link_options_shared_lib(executorch)
 # Real integrations should supply their own YAML file that only lists the
 # operators necessary for the models that will run.
 #
+if(NOT EXECUTORCH_BUILD_CADENCE)
```

Comment:
Rather than guarding this by `NOT EXECUTORCH_BUILD_CADENCE` flags i think we should maybe add a flag to not build portable ops. e.g. `BUILD_EXECUTORCH_PORTABLE_OPS` which is on by default and for cadence builds it can be disabled explicitly.

Reply:
updated the PR as per this suggestion